helm-charts/splunk-otel-collector/templates/secret-splunk.yaml

```diff
@@ -17,13 +17,13 @@ data:
   {{- if (eq (include "splunk-otel-collector.splunkPlatformEnabled" .) "true") }}
   splunk_platform_hec_token: {{ .Values.splunkPlatform.token | b64enc }}
   {{- end }}
-  {{- with .Values.splunkPlatform.clientCert }}
+  {{- with .Values.splunkPlatform.clientCert | default nil }}
```

Comment:
Should be `| default ""` as it is a default value in values.yaml for all of these.

Reply:
Thanks for the addition, fixed, packaged locally and test in a call with on demand case

Reply:
Ok, looks good. Still missing the changelog though